intuita-webview/src/campaignManager/TreeView/index.tsx

```diff
@@ -0,0 +1,154 @@
+import { ReactNode, useCallback, useState } from 'react';
+import Tree from '../../shared/Tree';
+import TreeItem from '../TreeItem';
+import {
+	Command,
+	TreeNode,
+} from '../../../../src/components/webview/webviewEvents';
+import { ReactComponent as BlueLightBulbIcon } from '../../assets/bluelightbulb.svg';
+import { ReactComponent as CaseIcon } from '../../assets/case.svg';
+import { ReactComponent as WrenchIcon } from '../../assets/wrench.svg';
+import { vscode } from '../../shared/utilities/vscode';
+import styles from './style.module.css';
+import cn from 'classnames';
+
+type Props = {
+	node: TreeNode;
+};
+
+const getIcon = (iconName: string | null, open: boolean): ReactNode => {
```

Comment:
wasn't this copied copied from somewhere?

Reply:
addressed